app/controllers/tasks_controller.rb

```diff
@@ -34,12 +28,17 @@ def create
 
   def update
     @task.update(task_params)
-    redirect_to tasks_path
+    redirect_to dashboard_path
   end
 
   def destroy
     @task.destroy
-    redirect_to tasks_path
+    redirect_to dashboard_path
+  end
+
+  def delete
```

Comment:
The REST `DELETE` verb should actually call the `destroy` method above, so you shouldn't need to have this method.